projects/distributed-tracing/src/shared/dashboard/interaction/detail-sheet/container/detail-sheet-interaction-container.component.ts

```diff
@@ -0,0 +1,12 @@
+import { ChangeDetectionStrategy, Component, Inject, InjectionToken } from '@angular/core';
+
+@Component({
+  selector: 'htc-detail-sheet-interaction-container',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `<ng-container [hdaDashboardModel]="this.detailModel"></ng-container>`
```

Comment:
ooh haven't seen the attribute directive used directly in so long!

Reply:
Yeah. We may add more features to this container which would be configurable through the interaction model.